tests/integration/config/test.yaml

```
@@ -0,0 +1,93 @@
+
+    datasets:
+      train:
+        _target_: fmcib.datasets.SSLRadiomicsDataset
+        path: "./data/dummy.csv" # Note: Change path
+        label: "label" # Note: Change to "malignancy" for Task 2 and "survival" for Task 3
+        radius: 25
+        orient: False # Note: Set orient to False for task 2 and task 3
+        resample_spacing: [1, 1, 1]
+        enable_negatives: False
+        transform:
+            _target_: monai.transforms.Compose
```

yamllint

[warning] wrong indentation: expected 10 but found 12

(indentation)